backends-common/cassandra/src/main/java/org/apache/james/backends/cassandra/init/configuration/ClusterConfiguration.java

```diff
@@ -81,6 +83,11 @@ public Builder hosts(Host... hosts) {
             return this;
         }
 
+        public Builder createKeyspace(boolean createKeyspace) {
```

Comment:
boolean arguments are evil and in this case, it's not necessary, you can have createKeyspace without any argument.